docs/faq.md

```diff
@@ -8,23 +8,16 @@ name = "spam"
 version = "2020.0.0"
 ```
 
-Commitizen **won't** use the `project.version` as a source of truth because it's a
-tool aimed for any kind of project.
-
-If we were to use it, it would increase the complexity of the tool. Also why
-wouldn't we support other project files like `cargo.toml` or `package.json`?
-
-Instead of supporting all the different project files, you can use `version_files`
-inside `[tool.commitizen]`, and it will cheaply keep any of these project files in sync
+Commitizen provides a [`pep621` version provider](config.md#version-providers) to get and set version from this field.
+You just need to set the proper `version_provider` setting:
 
 ```toml
+[project]
+name = "spam"
+version = "2020.0.0"
```

Comment:
do we support this kind of version?

Reply:
I should change the example. `2020` is an integer, so `commitizen` can work with this. The problem is with `calver` version management, not really specified (each project having its own rules) and I don't want people to think this is properly supported just with this example.
Note: but adding `calver` support by using the same pattern for a `VersionSchemProvider` could be a good idea for later

Reply:
Version changed to something not looking like CalVer 👍🏼